### ISSUE TITLE
Unsure about meaning of Doc in Levenshtein.java

### ISSUE THREAD

**Issue by samzmann** [LaVielle/swqc_final#1]
Note sure what you mean on line 64-67

```
	 * Dictionary sorting a single query which included multiple words.
	 * <p>
	 * Considering the query is combine with multiple words, and the result will be affected by 
	 * the order of words. It also the pre-load queries function from original Dataset csv.
```

Please explain and I can rephrase it to make it more understandable.

**Comment by wolf1991** [comment]
For queries, most of them are combine with many words.
I resorted them and combine to a long string for preparing to compare with others.
